services/autoscaling/src/simcore_service_autoscaling/models.py

```diff
@@ -62,3 +67,84 @@ class Cluster:
 class DaskTask:
     task_id: DaskTaskId
     required_resources: DaskTaskResources
+
+
+class EC2InstanceBootSpecific(BaseModel):
+    ami_id: str
+    custom_boot_scripts: list[str] = Field(
```

Comment:
MINOR: perhaps `TypeAlias` many of the `str` around for doc? e.g. in this case `CommandStr: Typealias = str`

Reply:
done